test/sanity/pep8/legacy-files.txt

```diff
@@ -1,3 +1,8 @@
+hacking/conf2yaml.py
+lib/ansible/constants.py
+lib/ansible/cli/config.py
+lib/ansible/config/data.py
+lib/ansible/config/manager.py
```

Comment:
Err.. which means that you need to pull those changes into your PR, no?  just like you'd pull a logic bug fix into your PR rather than overwriting it with the old code in different files.